### PR TITLE
Unify a set of common pseudo-instructions across codegens

### PR DESCRIPTION
**Pull request by fjeremic**
See each individual commit for which pseudo-instructions were unified. I did not touch x86 or ARM because they need to be taught to use `TR::InstOpCode` first and that is a much larger piece of work than what is done here. I will be tackling this once this PR is merged.

Depends on eclipse/omr#6000

### CODE REVIEW

**Comment by fjeremic** [comment]
Jenkins build all jdk8,jdk11 depends eclipse/omr#6000

**Comment by fjeremic** [comment]
Jenkins test sanity all jdk8,jdk11 depends eclipse/omr#6000

**Comment by fjeremic** [comment]
@dsouzai do you want to be a committer for this one?

**Review by dsouzai — APPROVED** [review]

**Comment by dsouzai** [comment]
Not sure what's going on with the build jobs; looks unrelated to the changes, but I would prefer to see a clean build only because it will ensure that all tests are run; seems like a lot of them got aborted.

**Comment by fjeremic** [comment]
Jenkins test sanity all jdk8,jdk11 depends eclipse/omr#6000

**Comment by fjeremic** [comment]
Looks like eclipse/omr#6009 needs to go in first to unblock the build here.

**Comment by fjeremic** [comment]
Jenkins test sanity s390x jdk8,jdk11 depends eclipse/omr#6000

**Comment by fjeremic** [comment]
Jenkins test sanity all jdk8,jdk11 depends eclipse/omr#6000

**Comment by genie-openj9** [comment]
Can one of the admins verify this patch?

**Comment by fjeremic** [comment]
@dsouzai what do we want to do about this one? We seem to be wrestling with infra at this point. Do we just keep retrying?

**Comment by fjeremic** [comment]
@dsouzai the two AIX failed tests seem to be because of #10919. I don't think this change could have caused it.

**Comment by dsouzai** [comment]
Yeah if I go to the actual test job there are no failures.

**Comment by fjeremic** [comment]
@dsouzai looks like OMR-Acceptance promoted and the openj9 branch is even with master right now. It seems like a good time for these changes to go in.

**Pull request merged**
